src/Backend/Lua/Builtin.hs

```diff
@@ -118,6 +118,30 @@ builtins =
           return var[1]
         end
       |] )
+  , ( vStrVal, "__builtin_strval", [], Nothing
```

Comment:
Ew, ew, ew.

Reply:
You come up with a better way to do it, then :/